vnext/Microsoft.ReactNative/ReactHost/React.h

```diff
@@ -22,7 +22,12 @@
 #undef GetCurrentTime
 #endif
 
+#ifndef CORE_ABI
+// The IReactInstance.h brings dependency on XAML. Exclude it for the UI technology independent code.
```

Comment:
non-blocking for this PR, but we should split ireactinstance into UI-dependent/independent (see e.g. what I did for iviewmanager in #5642)

Reply:
Agree. The IReactInstance.h is empty. It must be just removed. While the rest of the code must be clearly separated between XAML-dependent and XAML-independent. In Windows we must have ability using different UI technologies for RNW.